app/Urls.go

```diff
@@ -1,18 +0,0 @@
-package app
-
-import (
-	"example/persons/Control"
-	"example/persons/Control/CreateUser"
-	"example/persons/Control/DeleteUser"
-	"example/persons/Control/GetUser"
-	"example/persons/Control/GetUsers"
-	"example/persons/middle"
-)
-
-func Urls() {
-	router.POST("/CreateUsers",middle.Authenticate, CreateUser.CreateUserHandler)
```

Comment:
try not to pass middleware function in request and use it by itself

Reply:
done